mobile/src/bundles/users/components/image-picker/image-picker.tsx

```diff
@@ -4,59 +4,73 @@ import { type StyleProp, type ViewStyle } from 'react-native';
 import { launchCamera, launchImageLibrary } from 'react-native-image-picker';
 import { type ImagePickerResponse } from 'react-native-image-picker';
 
-import { Button } from '~/bundles/common/components/components';
+import { Button, Modal } from '~/bundles/common/components/components';
 import { ButtonType } from '~/bundles/common/enums/enums';
+import { useCallback, useState } from '~/bundles/common/hooks/hooks';
 
 type ImagePickerProperties = {
     label: string;
+    onImageLoad: (payload: Promise<ImagePickerResponse>) => void;
     containerStyle?: StyleProp<ViewStyle>;
 };
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const getImageFromLibrary = async (): Promise<ImagePickerResponse> => {
-    return await launchImageLibrary({ mediaType: 'photo' });
-};
+const ImagePicker: React.FC<ImagePickerProperties> = ({
+    label,
+    onImageLoad,
+    containerStyle,
+}) => {
+    const [isPopUpActive, setIsPopUpActive] = useState(false);
+
+    const getImageFromLibrary = useCallback((): void => {
+        setIsPopUpActive(false);
```

Comment:
Do you need to `setIsPopUpActive(false)` since `isPopUpActive` is `false` by default?

Reply:
It's for closing pop up element, after user choose an option from pop up

Reply:
I see